### PR TITLE
added samtools to build and test

### PR DESCRIPTION
**Pull request by asafpr**
Update rilseq, samtools issues
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by asafpr** [comment]
@BiocondaBot please fetch artifacts

**Comment by asafpr** [comment]
@BiocondaBot please add label

File: recipes/rilseq/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RILseq-{{ version }}.tar.gz"
-  sha256: 969a009f06c4cfcfeb59c5c9f0680e2196b77a55b663fb459f97c0e5cdd3e02a
+  sha256: 7386116c01b399e1c76f6affd8a75d151ab8e10bee70d2852baabffa35b3e8b1
 
 build:
-  number: 0
+  number: 1
```
Comment (nsoranzo) [inline recipes/rilseq/meta.yaml:13]: This should stay at 0, since you are increasing the version.

Comment (asafpr) [reply recipes/rilseq/meta.yaml:13]: There's already a 0.78 version that was automatically generated

File: recipes/rilseq/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
-    - python
+    - python >=3
     - scipy
     - bwa >=0.7.12
-    - samtools >=1.2
+    - samtools >=1.9
 
 test:
+  commands:
+    - samtools 2>&1 | grep "Usage"
```
Comment (nsoranzo) [inline recipes/rilseq/meta.yaml:32]:
```suggestion
    - samtools --help
```

samtools recipe use this, so I suppose it may be safer in the long run.

**Comment by asafpr** [comment]
@BiocondaBot please merge

**Pull request merged**